### ISSUE TITLE
[New Port Request] dbcppp

### ISSUE THREAD

**Issue by stvsl** [microsoft/vcpkg#40953]
### Library name

dbcppp

### Library description

A C/C++ DBC file parser based on boost.spirit. This library is designed for decoding performance.

### Source repository URL

https://github.com/xR3b0rn/dbcppp.git

**Comment by rtzoeller** [comment]
Agreed this would be useful; I tried to implement it but ran into some CMake issues with this specific dependency.
